libqtile/utils.py

```diff
@@ -199,6 +200,21 @@ def get_cache_dir() -> str:
         os.makedirs(cache_directory)
     return cache_directory
 
+def _get_config_file():
+    config_home = os.getenv("XDG_CONFIG_HOME", "~/.config")
+    if (config_file:= Path(f"{config_home}/qtile/config.py")).exists():
```

Comment:
Does this need the tilde resolved before the `exists`?
```suggestion
    config_home = Path(os.getenv("XDG_CONFIG_HOME", "~/.config")).expanduser()
    if (config_file:= config_home / "qtile/config.py").exists():
```

Reply:
Yes it needs the tilde resolved before the `exists`

fixed here: https://github.com/qtile/qtile/pull/4490/commits/8b38ef38099d77b3bb96fb7d76e9832c7d724b4e